cmd/atlas/internal/cmdapi/migrate.go

```diff
@@ -53,6 +53,8 @@ const (
 	migrateApplyFromVersion     = "from"
 	migrateApplyBaselineVersion = "baseline"
 	migrateApplyTxMode          = "tx-mode"
+	migrateFlagImportFrom       = "from"
+	migrateFlagImportTo         = "to"
```

Comment:
You can use `migrateFlagTo`. I'd say, move the `from` to be shared flag as well.